Proposal.md

```diff
@@ -9,16 +9,17 @@
 
 * _October 24, 2019:_ Renamed the `move(...)` methods to `gather(...)` for multiple-range moves and `shift(from:toJustBefore:)` for single-range and single-element moves to better reflect their behavior, and removed the `move(from:to:)` method. Added `elements(within:)` method for getting the individual index values in a `RangeSet` when the `Bound` type isn't integer-strideable.
 * _October 31, 2019:_ Removed `SetAlgebra` conformance and the `Elements` collection view, as `RangeSet` can't guarantee correct semantics for individual index operations without the parent collection. Renamed `elements(within:)` to `individualIndices(within:)`. Deferred the rotating and partitioning methods to a future pitch.
+* _November 6, 2019:_ Removed the `individualIndices(within:)` method. Simplified argument labels for the `shift` and `gather` methods. Revised the proposal to make the relation between individual indices and the ranges in a range set more clear, among other things. Renamed `IndexingCollection` to `DiscontiguousSlice`, and made it its own slice type.
 
 ## Introduction
 
-We can use a range to address a single range of consecutive elements in a collection, but the standard library doesn't currently provide a way to access discontiguous elements. This proposes the addition of a `RangeSet` type that can store the location of any number of collections indices, along with collection operations that let us use a range set to access or modify the collection.
+We can use a `Range<Index>` to refer to a group of consecutive positions in a collection, but the standard library doesn't currently provide a way to refer to discontiguous positions in an arbitrary collection. I propose the addition of a `RangeSet` type that can store any number of positions, along with collection algorithms that operate on those positions.
```

Comment:
Yeah, we need to fix this; it can't store any positions.  It stores ranges, which can be used to filter indices.